app/elements/candidates/invite-candidate-input.html

```diff
@@ -76,7 +67,30 @@
 			_removeItem: function(){
 				this.candidate.Removed = true;
 				this.remove();
-			}
+			},
+            _afterProperties: function(challenges, candidate){
```

Comment:
will fail at check because you don` use challenges or candidate make it `_afterProperties:function(){...}`